build.gradle

```diff
@@ -157,13 +158,19 @@ task copyDeps(type: Copy) {
     rename '^(.*)(-[0-9]+[.[0-9]+]+(-SNAPSHOT)?)(.*)$', '$1$4'
 }
 
-runIde {
-    jvmArgumentProviders.add({
-        [
-                "--add-exports",
-                "java.base/jdk.internal.vm=ALL-UNNAMED",
-        ]
-    } as CommandLineArgumentProvider)
+intellijPlatformTesting.runIde {
+    runIdeLocally {
+        version = providers.gradleProperty("runIdeUiversion")
```

Comment:
Instead of using runIdeUiVersion, could you provide a more generic name that would be suitable for both the runIde and runIdeForUiTests tasks?